### PR TITLE
Feature/204 swagger UI page

### PR DESCRIPTION
**Pull request by maxdiebold-erg**
## Related Issues:
* [EQ-204](https://jira.epa.gov/browse/EQ-204)

## Main Changes:
* Added Swagger API documentation for the profile `/data` routes and there corresponding `/count` routes.
  * I left the contact email blank for now. Also most parameters still need descriptions, since we're still waiting on glossary/tooltip descriptions from EPA.
  * The configuration file is stored with the S3 content, so the filepath shown in the UI would be strange if displayed:
  
swagger

If we want this JSON file to be publicly accessible, we may need to move it to the `public` folder; currently, the link to the file is hidden.

## Steps To Test:
1. Go to the API docs page: http://localhost:3000/api-docs.
2. Confirm that all routes are properly documented, and that trying queries works as expected.

### CODE REVIEW

**Review by cschwinderg — CHANGES_REQUESTED** [review]
Looks good! I haven't looked to closely at the openapi file yet. 

Can you add a button next to the Glossary and National Downloads buttons that opens the api documentation?

Are you getting these console errors/warnings on your end when you run the app?
image

File: etl/app/content-private/domainValueMappings.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
   "actionAgency": {
     "domainName": "AgencyCode"
+    "valueField": "name"
```
[the remark this answers is not in the document]
Comment (maxdiebold-erg) [reply etl/app/content-private/domainValueMappings.json]: Thanks!

**Comment by maxdiebold-erg** [comment]
I am getting those warnings, I'll try checking those repositories to see if anybody else is having those issues.

**Comment by maxdiebold-erg** [comment]
It looks like the warnings are due to the fact that the _Autolinker_ package is generating source maps when compiling their TypeScript source code, but they aren't including the source code in the NPM package (as raised in https://github.com/gregjacobs/Autolinker.js/issues/396 and discussed generally in https://github.com/facebook/create-react-app/discussions/11767). It doesn't look like there is a way to disable this warning for a particular package, but it would only become a problem when trying to debug _Autolinker_ code.

Both `swagger-ui` and `swagger-ui-react` include `remarkable` as a dependency, which includes `autolinker`, so I think the only way to avoid the warning would be to keep the bundles in the `public` folder, like LEW does, but that option is a lot heavier.

**Comment by cschwinderg** [comment]
> Both `swagger-ui` and `swagger-ui-react` include `remarkable` as a dependency, which includes `autolinker`, so I think the only way to avoid the warning would be to keep the bundles in the `public` folder, like LEW does, but that option is a lot heavier.

 We may need to keep the bundles in the `public` folder just to cut down on the noise in the console. We can discuss this lader though. I'll go ahead and merge this in as is.

**Review by cschwinderg — APPROVED** [review]

**Pull request merged**